lib/ZwaveClient.js

```diff
@@ -1889,6 +1889,43 @@ ZwaveClient.prototype.callApi = async function (apiName, ...args) {
   return result
 }
 
+/**
+ * Send broadcast write request
+ *
+ * @param {Object} valueId Zwave valueId object
+ * @param {Integer|String} value The value to send
+ */
+ZwaveClient.prototype.writeBroadcast = async function (valueId, value) {
+  if (this.driverReady) {
+    try {
+      const broadcastNode = this.driver.controller.getBroadcastNode()
+
+      await broadcastNode.setValue(valueId, value)
+    } catch (error) {
+      debug(`Error while sending broadcast ${value} to CC ${valueId.commandClass} ${valueId.property} ${valueId.propertyKey || ''}: ${error.message}`)
+    }
+  }
+}
+
+/**
+ * Send multicast write request to a group of nodes
+ *
+ * @param {number[]} nodes Array of nodes ids
+ * @param {Object} valueId Zwave valueId object
+ * @param {Integer|String} value The value to send
+ */
+ZwaveClient.prototype.writeMulticast = async function (nodes, valueId, value) {
+  if (this.driverReady) {
+    try {
+      const multicastGroup = this.driver.controller.getMulticastGroup(nodes)
+
+      await multicastGroup.setValue(valueId, value)
```

Comment:
Like I wrote in my comment, I would check for `ZWaveErrorCodes.CC_NotSupported` and fall back to iterating singlecast requests.

Reply:
@AlCalzone Could this cause multiple writes?

Reply:
No, the checks happen before anything is sent.

Reply:
Done